composer.json

```diff
@@ -17,10 +17,10 @@
     "require": {
         "php": "^7.4|^8.0",
         "google/apiclient": "^2.7",
-        "tipoff/support": "^1.1.2"
+        "tipoff/support": "^1.2.6"
     },
     "require-dev": {
-        "tipoff/test-support": "^1.0"
+        "tipoff/test-support": "^1.0.3"
```

Comment:
Thanks for bumping to the current releases!

Reply:
👍  Sorry I didn't know about it before